docs/minio.md

```diff
@@ -13,16 +13,16 @@ sidebar_label: Minio
 
 <br>
 
-Cloud native Object storage is easy to deploy with Minio and OpenEBS. Both run in user space and do not have any kernel dependencies. Minio and OpenEBS start with a very simple configuration of few gigabytes and scale on demand to very high capacities.
+Cloud native Object storage is easy to deploy with Minio and OpenEBS. Both run in user space and do not have any kernel dependencies. Minio and OpenEBS start with a very simple configuration of few gigabytes and scale on demand to high capacity.
```

Comment:
I am sending a new PR with using all storage engine by this week. So we can ignore the usage of Minio . Actual usage is MinIO